### ISSUE TITLE
Training hyperparameters

### ISSUE THREAD

**Issue by Zhimin00** [kvuong2711/aerial-megadepth#5]
Thanks for your awesome work. Can you provide the detailed training hyperparameters for finetuning DUSt3R and MASt3R?

**Comment by kvuong2711** [comment]
Hi @Zhimin00,

I added some information in finetuning [here](url). It mostly follows DUSt3R/MASt3R original settings, just minor changes in learning rate (depending on your GPUs/batch size you might need to scale the lr up or down). I trained on 8xA6000 => with `--batch_size=2 --accum_iter=2`, the effective batch size is 32.

**Comment by kvuong2711** [comment]
Closing for now. Feel free to re-open the issue if you run into any issue finetuning the model.

**Comment by ayushzenith** [comment]
Hey Khiem, quick questions: 
Could you possibly share your full command that you ran? (That could help me a lot in reproducing your training results!)
    - Did you use the same training critereon as the original authors or did you modify it?
    - Did you use any of the original datasets and if so did you stick with the original splits?

**Comment by hectorcarrion** [comment]
can we reopen the issue with the above comment?
